runtime/manifest.js

```diff
@@ -143,8 +144,17 @@ class Manifest {
     let tags = options.tags || [];
     let subtype = options.subtype || false;
     function typePredicate(view) {
+      let resolvedType = type;
+      let primitiveType = type.isSetView ? type.primitiveType() : type;
+      if (primitiveType.isVariable) {
+        if (!primitiveType.data.isResolved) {
```

Comment:
then this is might be a new `type.isResolved()` that suggests whether all contained type variables have been resolved

Reply:
done (at least, i hope, this was what you meant ))